core/src/main/java/org/pentaho/di/connections/vfs/VFSConnectionDetails.java

```diff
@@ -2,7 +2,7 @@
  *
  * Pentaho Data Integration
  *
- * Copyright (C) 2019 by Hitachi Vantara : http://www.pentaho.com
+ * Copyright (C) 2024 by Hitachi Vantara : http://www.pentaho.com
```

Comment:
this copyright date has to be `2019 - 2024`

example: https://github.com/pentaho/pentaho-kettle/blob/3880e41dedd99a7b2917fe9479f4fc171f8dfbed/core/src/main/java/org/pentaho/di/connections/ConnectionDetails.java#L5